examples/engine/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/nspcc-dev/neo-go/examples/engine
+
+go 1.17
```

Comment:
We currently require 1.15, maybe it's more appropriate here also?

Reply:
Yeah, fixed.